src/main/java/offeneBibel/parser/OffeneBibelParser.java

```diff
@@ -503,7 +546,9 @@ public Rule Alternative() {
             push(new ObAstNode(ObAstNode.NodeType.alternative)),
             OneOrMore(FirstOf(
                 ScriptureText(),
+                Sequence("<s>", createOrAppendTextNode(match()), ScriptureText(), "</s>", createOrAppendTextNode(match())),
                 Quote(),
+                InnerQuote(),
```

Comment:
Weiß auch nicht mehr wo, aber auf der Seite war schon ein äußeres Quote, dann kam aber eine Alternative, in der sich das innere Quote befand. Und vor ich jetzt eine AlternativeInsideQuote schaffe und alles dupliziere (oder mich darüber informiere, ob Parboiled irgendwie State-Variablen beim Parsen nutzen kann), war das die einfachere "Lösung".